common/src/config.rs

```diff
@@ -1,4 +1,4 @@
 pub const GENERATION_INPUTS_DEFAULT_OUTPUT_DIR: &str = "generation_inputs";
-pub const MAIN_TEST_DIR: &str = "BlockchainTests";
+pub const MAIN_TEST_DIR: &str = "Cancun/BlockchainTests";
```

Comment:
The name may be confusing, but it's implying all tests until Cancun included.

Reply:
Should we add a comment to explain what `MAIN_TEST_DIR` corresponds to then?

Reply:
Yeah it might be worth a comment just explaining that a sub-group is now the main sub-group, but I'm good either way.